python_modules/dagster/dagster_tests/cli_tests/command_tests/test_cli_commands.py

```diff
@@ -563,3 +564,10 @@ def test_run_wipe_incorrect_delete_message():
     result = runner.invoke(run_wipe_command, input="WRONG\n")
     assert "Exiting without deleting all run history and event logs" in result.output
     assert result.exit_code == 0
+
+
+def test_use_env_vars_for_cli_option():
+    runner = CliRunner(env={"DAGSTER_CLI_ASSET_WIPE_ALL": "1"})
+    result = runner.invoke(cli, ["asset", "wipe"], auto_envvar_prefix=ENV_PREFIX, input="DELETE\n")
+    assert "Removed asset indexes from event logs" in result.output
+    assert result.exit_code == 0
```

Comment:
this needs to be gated with `instance_for_test` context manager like ive done in https://dagster.phacility.com/D4783 to prevent running this test from deleting your actual instance DB (which in its current form, it will do)

Reply:
Thanks for the tips. I was not familiar with dagster internal, so I didn't know that it will involve the instance DB.
 just fixed it!

Reply:
I tried to apply the context manager as you suggested, but the test fails with

```bash
AssertionError: assert 'Removed asset indexes from event logs' in 'Usage: group asset wipe [OPTIONS] [KEY]...\n\nError: Error, configured Dagster instance does not have asset aware event storage.\n'
```

Reply:
After some debugging: I found that the instance created by `instance_for_test` is `SqliteEventLogStorage`, which is not a subclass of `AssetAwareEventLogStorage`. 
Without the context manager, I think (not 100% sure), the instance created is a in memory instance `InstanceType.EPHEMERAL` with `event_storage=InMemoryEventLogStorage`., which is a subclass of `AssetAwareEventLogStorage`.

I wonder if without the context manager, is it ok to just use the in-memory instance? it's temporarily anyway, so the event storage will be destroyed even if we don't explicitly destroy it.

Reply:
how about testing a different CLI - maybe `dagster --version` ?

Reply:
@alangenfeld cilck behave a bit interesting with respect to group-level option, especially `version_option`. I submitted an Issue to upstream https://github.com/pallets/click/issues/1694. Until it's fixed, I think we can use the `debug` subcommand to trigger the `--version` behavior for testing purpose.